### PR TITLE
Add required labels to selectable d2l-list-items.

### PR DESCRIPTION
**Pull request by dbatiste**
This PR adds labels to selectable d2l-list-items that will be required in the near future in order to support changes that are being made to lists for selection, multi-selection, and multi-actions,

### CODE REVIEW

**Review by johngwilkinson — APPROVED** [review]
It's possible this component won't be released again in the near future - is it important to get this update into production by a certain time?

**Comment by dbatiste** [comment]
> It's possible this component won't be released again in the near future - is it important to get this update into production by a certain time?

Yeah I think we'll need this updated to unblock some core changes related to how labels are handled for list-items.  It looks like this doesn't use the semantic release github action, do I think I will need to manually update the version and create the release.

**Review by bemailloux — APPROVED** [review]

**Comment by bemailloux** [comment]
> I think I will need to manually update the version and create the release

Our release process is documented here: https://github.com/Brightspace/insights-engagement-dashboard#releasing-and-versioning

TL;DR if this is going into 20.21.7 it should be released as 1.2107.x
